matter/src/pairing/vendor_identifiers.rs

```diff
@@ -0,0 +1,16 @@
+#[repr(u16)]
+pub enum VendorId {
+    CommonOrUnspecified = 0x0000,
```

Comment:
Any particular reason to keep the vendor specific codes around? How about we just retain the ones that are used in `is_vendor_id_valid_operationally()` ?

Otherwise, this list will keep growing

Reply:
Agreed.